helm/mergeable/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for mergeable.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+namespace:
+  enabled: true
+  annotations: {}
+  labels: {}
+
+image:
+  repository: "DOCKER_REPO/mergeable"
```

Comment:
Is DOCKER_REPO suppose to be an env variable?

Reply:
it's supposed to be filled, no need to get from environment values I guess, but just static typing